File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool: 
        """
         intiakize hashset set()
         if num is in hashset return true 
         else add to hashset 
         if no duplicates return false 
        
        """ 
        seen = set()

        for num in nums:
          if num in seen:
            return True 

          else:
            seen.add(num)

        return False 

File: 0217-contains-duplicate/0217-contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
         Set<Integer> seen = new HashSet<>();
         for (int num:nums){
            if (!seen.contains(num)){
                seen.add(num);
            } else {
                return true;
            }
         }
         return false;
    }
}